### PR TITLE
Install wget in Cloud Docker images

### PR DESCRIPTION
**Pull request by pugnascotia**
Cloud needs `wget` to be available in 8.x, so add it to their Docker images.

### CODE REVIEW

File: qa/os/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java [hunk 1]
```diff
@@ -480,6 +480,15 @@ private static void verifyCloudContainerInstallation(Installation es) {
         } else {
             assertThat("Cloud image should not have any plugins in " + pluginArchive, plugins, empty());
         }
+
+        // Cloud uses `wget` to install plugins / bundles.
+        Stream.of("wget")
```
Comment (mark-vieira) [inline qa/os/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java:485]: So they require both `curl` _and_ `wget`?

**Review by mark-vieira — APPROVED** [review]
:+1:

File: qa/os/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java (hunk 1 above)
Comment (pugnascotia) [reply qa/os/src/test/java/org/elasticsearch/packaging/util/docker/Docker.java:485]: ...yes.

**Comment by pugnascotia** [comment]
Backported to 8.0 in 9c8fd904ca8

**Comment by pugnascotia** [comment]
Backported to `7.16` in 9aea490eaeb.

**Pull request merged**